pom.xml

```diff
@@ -802,20 +802,6 @@ under the License.
 						</executions>
 					</plugin>
 				</plugins>
-				<pluginManagement>
```

Comment:
As mentioned above, it's not actually needed for our current release process but I'd like to make it usable for that. Apparently how you use it with the Apache parent pom is to put it into the `pluginManagement` section.